hardware_interface/src/fake_components/generic_system.cpp

```diff
@@ -273,6 +298,25 @@ std::vector<hardware_interface::CommandInterface> GenericSystem::export_command_
 {
   std::vector<hardware_interface::CommandInterface> command_interfaces;
 
+  auto populate_command_interfaces = [this](
```

Comment:
Can we somehow merge `populate_command_interfaces` and `populate_state_interfaces`? Why we have this two separated?

Reply:
Done. But facing that test error for command interface count. I am checking that.